Jenkinsfile

```diff
@@ -87,6 +87,8 @@ pipeline {
                                   -e DOCKER_EXPERIMENTAL \
                                   -e DOCKER_GITCOMMIT=${GIT_COMMIT} \
                                   -e DOCKER_GRAPHDRIVER \
+                                  -e VALIDATE_REPO=${GIT_URL} \
+                                  -e VALIDATE_BRANCH=${CHANGE_TARGET} \
```

Comment:
FWIW; I tried setting these as global variable in the Jenkinsfile, but the Git plugin does not expose it's variables at that point, so while we can get `CHANGE_TARGET`, we can't get `GIT_URL` (without additional hackery), so I went for just passing them here inline

Reply:
To clarify; I assumed Jenkins would set that env-var for the whole job, but the Git plugin only sets it _after_ if does the checkout, so it's only present in the stages itself.

My initial attempt was to map those env-vars in the top level block;

```
    environment {
        DOCKER_BUILDKIT     = '1'
        DOCKER_EXPERIMENTAL = '1'
        DOCKER_GRAPHDRIVER  = 'overlay2'
        APT_MIRROR          = 'cdn-fastly.deb.debian.org'
        CHECK_CONFIG_COMMIT = '78405559cfe5987174aa2cb6463b9b2c1b917255'
        TESTDEBUG           = '0'
        TIMEOUT             = '120m'
        TIMEOUT             = '120m'
        TIMEOUT             = '120m'
        VALIDATE_REPO       = "${env.GIT_URL}"
        VALIDATE_BRANCH     = "${env.CHANGE_TARGET}""
    }
```

Which worked for `$CHANGE_TARGET` / `VALIDATE_BRANCH`, but `VALIDATE_REPO` would always be `NULL`